src/odemis/driver/hamamatsurx.py

```diff
@@ -595,9 +627,24 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         # read-only VAs
         # TODO: Trig. Mode, Trig. level, Trig. slope??? plus MD!?
 
-    def _updateStreakMode(self, value):
+    def GetStreakMode(self):
         """
-        update the mode VA
+        Get the current value from the the streak unit HW.
+        :return: (bool) current streak mode value
+        """
+        streakMode_raw = self.parent.DevParamGet(self.location, "Mode")  # returns a list
+        if streakMode_raw[0] == "Focus":
+            streakMode = False
+        else:
```

Comment:
You could make it extra careful by writting:
```
elif streakMode_raw[0] == "Operate":
    streakMode = True
else:
    logging.warning("Unexpected streak mode %s", streakMode_raw)
    streakMode = False  # safer?
```

Reply:
I think it will never return anything different then "Focus" or "Operate".

Reply:
You should never trust a hardware from _always_ doing what's documented.

Indeed, _probably_ it never returns anything else, but you can't be 100% certain. Maybe there is a network error and you receive only "Oper", or maybe, in the future, Hamamatsu will decide to update their software and it'll report "Focused", or "operate". Then it's nice to immediately get some warning.

Reply:
ok convinced :)